vignettes/corrplot-intro.Rmd

```diff
@@ -289,6 +289,25 @@ corrplot(M, p.mat = res1[[1]], low = res1[[2]], upp = res1[[3]],
          addrect = 3, rect.col = "navy", plotC = "rect",cl.pos = "n")
 ```
 
+```{r ci_with_laber}
+# Hmisc::rcorr provides a p-value matrix, but we don't want to introduce the dependency
+# HM <- Hmisc::rcorr(as.matrix(mtcars))
+
+# therefore, we use fake p-values
```

Comment:
It is confused to statisticians with fake p-values . Please try `cor.mtest()`  function in example code of `corrplot()`?

Reply:
ok

Reply:
I tried to use a simplified `cor.mtest` function as follows:
```r
cor.mtest <- function(mat, conf.level = 0.95){
  mat <- as.matrix(mat)
  n <- ncol(mat)
  p.mat <- matrix(NA, n, n)
  diag(p.mat) <- NA
  for (i in 1:(n - 1)) {
    for (j in (i + 1):n) {
      tmp <- cor.test(mat[,i], mat[,j], conf.level = conf.level)
      p.mat[i,j] <- p.mat[j,i] <- tmp$p.value
    }
  }
  p.mat
}
```

Reply:
Two `cor.mtest ()` in `corrplot-intro.Rmd`. Line 327 and 242.

Reply:
I mean README.md figure did not display on home page.